src/main/java/mate/academy/spring/mapper/impl/response/UserResponseMapper.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.spring.mapper.impl.response;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserResponseMapper implements DtoResponseMapper<UserResponseDto, User> {
+    @Override
+    public UserResponseDto toDto(User user) {
+        UserResponseDto userResponseDto = new UserResponseDto();
+        userResponseDto.setEmail(user.getEmail());
```

Comment:
```suggestion
        UserResponseDto userResponseDto = new UserResponseDto();
        userResponseDto.setId(user.getId());
        userResponseDto.setEmail(user.getEmail());
```
i think here also need to get the id of user